components/Navbar.vue

```diff
@@ -59,15 +59,90 @@
       <MobileExpandableSection v-if="isMobile" :title="$t('explore')">
         <NavbarExploreOptions />
       </MobileExpandableSection>
+
+      <MobileExpandableSection
+        v-if="isMobile"
+        :no-padding="true"
+        :title="$t('create')">
+        <b-navbar-item
+          data-cy="classic"
+          :to="`/${urlPrefix}/create`"
+          tag="nuxt-link">
+          {{ $t('classic') }}
+        </b-navbar-item>
+        <b-navbar-item
+          data-cy="simple"
+          :to="`/${urlPrefix}/mint`"
+          tag="nuxt-link">
+          {{ $t('simple') }}
+        </b-navbar-item>
+        <b-navbar-item
+          data-cy="creative"
+          :to="`/${urlPrefix}/creative`"
+          tag="nuxt-link">
+          {{ $t('creative') }}
+        </b-navbar-item>
+      </MobileExpandableSection>
```

Comment:
Could you pls put this part of code into the `CreateDropdown` component, because they are doing the same thing.

Reply:
done